### PR TITLE
updates forwarders to improve pod logs, adds low alert for forwarder errors

### PR DESCRIPTION
**Pull request by tonytheleg**
For:
[OSD-20974](https://issues.redhat.com//browse/OSD-20974)
[OSD-21059](https://issues.redhat.com//browse/OSD-21059)
[SDE-3870](https://issues.redhat.com//browse/SDE-3870) (PMR Epic)

Changes:
* Updates the forwarder images to include updates that add splunkd logs to pod log output to improve troubleshooting ([PR](https://github.com/openshift/splunk-forwarder-images/pull/25)
* Adds a low alert for when the Splunk forwarder is failing to aid in troubleshooting Splunk issues when logs are missing (part of PMR follow up)

Both updates have been tested with validation info in their respective cards above

### CODE REVIEW

File: hack/olm-registry/olm-artifacts-template.yaml [hunk 1]
```diff
@@ -266,6 +266,23 @@ objects:
...
+          - name: splunk-forwarder
+            rules:
+              - alert: SplunkForwarderComponentUnhealthy
+                annotations:
+                  message: One or more Splunk Forwarder Components are reporting unhealthy. This issue can impact log forwarding to Splunk, leading to log data loss.
+                expr: splunk_forwarder_component_unhealthy > 0
+                for: 15m
+                labels:
+                  severity: warning
+
```
Comment (mjlshen) [inline hack/olm-registry/olm-artifacts-template.yaml:285]: Honestly, I see how it makes sense here, but by convention I think we put these into MCC. What do you think?

**Review by mjlshen — COMMENTED** [review]
Changes make sense, non-blocking comment from me.

File: hack/olm-registry/olm-artifacts-template.yaml (hunk 1 above)
Comment (tonytheleg) [reply hack/olm-registry/olm-artifacts-template.yaml:285]: I wasn't sure if there was a pre-defined pattern for SREP to be fair. I've seen some put it in operator repo, some in MCC. Personally, id rather make it here with the operator as MCC is not automatically updated in FedRAMP, not heavily tested in FedRAMP, and has a higher change to impact our environment for such a small change. While many MCC changes are set to skip FedRAMP clusters its not always the case. 

I'm ok with moving it if there is a strong desire/concern but just my two cents. Im flexible.

Comment (mjlshen) [reply hack/olm-registry/olm-artifacts-template.yaml:285]: I'm ok either way too. The PR's already made, for me (as a non-approver) that's good enough reason to keep it here and minimize busy-work.

**Pull request merged**